tools/webpack-make.js

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env node
 
-import webpack from 'webpack';
+import child_process from 'child_process';
 import path from 'path';
-import argparse from 'argparse';
 import process from 'process';
-import { CockpitRsyncWebpackPlugin } from '../pkg/lib/cockpit-rsync-plugin.js';
+
+// ensure node_modules is present and up to date
+child_process.spawnSync('tools/node-modules', ['make_package_lock_json'], { stdio: 'inherit' });
```

Comment:
This is an example of a thing that doesn't need `package-lock.json`...

On the other hand, `checkout` will always check out.

We need some kind of a new approach/target here, I think...  But let's not block on it.